app/controllers/panel_controller.rb

```diff
@@ -62,7 +62,7 @@ def generate_db_token
     )
     results_validator.validate(competition_ids)
     render json: {
-      has_results: results_validator.has_results?,
+      has_results: results_validator.results?,
```

Comment:
Oof, close call. Only `results?` is like... Yeah, Results. What about them?

Or in other words, nothing about a `?` suffix inherently screams `has`. It might be a shorthand for `is_` or `can_` or `did_` as well FWIW. Any chance for a better rename without introducing `has_results` as a cop exception?

Reply:
(although for the sake of reviewing I will accept a cop exception as a last resort)

Reply:
any_results?